services/self-service/src/main/java/com/epam/dlab/backendapi/dao/OdahuDAO.java

```diff
@@ -39,4 +39,6 @@
     void updateStatus(String name, String project, String endpoint, UserInstanceStatus status);
 
     void updateStatusAndUrls(OdahuResult result, UserInstanceStatus status);
+
+    List<OdahuDTO> getOdahuProjectClusters(String project, String endpoint);
```

Comment:
1) Move it under the  **findOdahuClusters()** declaration.
2) Name it **findOdahuClusters**